packages/stream_video/CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## 0.4.3
+
+🐞 Fixed
+* Fixed reconnection flow.
+* Fixed compilation errors on web.
+* Fixed screen-sharing not working on some versions of Android.
+* Fixed accepting incoming call during another call (or outgoing call).
+* Fixed CallKit incomming push label and Android accept call buttons icons for audio-only calls.
+
+✅ Added
+* Added support for missed calls. Push notificationa and `CallMissedEvent` event will be send now when user misses the call.
+* Added support for call transcription [check transcription cookbook](https://getstream.io/video/docs/flutter/ui-cookbook/transcriptions/)
+    * Added `startTranscription()`, `stopTranscription()` and `listTranscriptions()` methods to `Call` class.
+* Added option to take a screenshot during the call [check screenshot documentation](https://getstream.io/video/docs/flutter/screenshots/)
+    * Added `takeScreenshot()` method to `Call` class.
+* Added `recordingExternalStorage` optional param to `call.startRecording()` method.
+* Added `requestScreenSharePermission()` method to `Call` class that ensures correct permissions are given for screen-sharing on Android.
+
+🔄 Changed
+* Changed `role` member to `roles`, to reflect posibility of user having mutliple roles, in `CallParticipantState` and `CallMember`.
+* Changed `SortParamRequest` to `SortParam`.
+
+🔄 Changed some of the signatures
+* `listRecordings()` method in PermissionsManager doesn't require `sessionId` param now.
+* `setParticipantPinned()` method in `Call` and `CallSession` now requires `sessionId`, `userId` and `pinned` params instead of `SetParticipantPinned` action.
```

Comment:
```suggestion
* `setParticipantPinned()` method in `Call` and `CallSession` now requires `sessionId`, `userId`, and `pinned` params instead of `SetParticipantPinned` action.
```